### PR TITLE
Feature/GLIDE-59

### PR DESCRIPTION
**Pull request by Minipeps**
Added 2-Turn-points detection and display on the map, as well as some useful informations about it.
The current detection algorithm complexity is `O(n^p)`, where n is the number of points of the track and p is the number of turn-points. It works for any value of `p>0`, but above 2 it's getting very slow to load the page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ The resulting web application will allow a user to import, manage and view detai
 
 ## Map view (WIP)
 
+![Map View](https://i.imgur.com/CYhubpg.png "Map View")
```
Comment (L-Le-Ninivin) [inline README.md]: Good doc.jpg

File: src/app/map-view/map-view.component.ts [hunk 2]
```diff
@@ -35,17 +40,23 @@ export class MapViewComponent implements OnInit {
...
   trackDay: string;
 
-  constructor(private mvs: MapViewService) { }
+  constructor(private map: MapViewService,
+              private parser: ParserService) { }
 
   ngOnInit() {
     // Bind variables
-    this.infosSubscription = this.mvs.infosSubject.subscribe(
+    this.infosSubscription = this.map.infosSubject.subscribe(
```
Comment (L-Le-Ninivin) [inline src/app/map-view/map-view.component.ts]: The change of name makes it clearer, thanks

File: src/app/services/map-view.service.ts [hunk 3]
```diff
@@ -58,31 +59,22 @@ export class MapViewService {
   minDelta = Infinity;
   maxDelta = -Infinity;
   deltaRange;
-  flightDuration;
-  totalDistance;
-  e2eDistance;
-  startAltitude;
-  stopAltitude;
-  highestPoint;
```
Comment (L-Le-Ninivin) [inline src/app/services/map-view.service.ts]: All of this went in the parser right?

File: src/app/services/parser.service.ts [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  flightDuration;
+  totalDistance;
+  e2eDistance;
+  startAltitude;
+  stopAltitude;
+  highestPoint;
+
+  constructor() {}
+
+  TurnPointsDetection(trackData, nPoints) {
```
Comment (L-Le-Ninivin) [inline src/app/services/parser.service.ts]: I know you spent a lot of time working on this algorithm, could you detail a bit how it works in comments?

Comment (Minipeps) [reply src/app/services/parser.service.ts]: Sure, good idea, will do

Comment (Minipeps) [reply src/app/services/parser.service.ts]: Done

**Pull request merged**